client/blocks/product-purchase-features-list/style.scss

```diff
@@ -13,8 +13,12 @@
 	padding-top: 12px;
 }
 
-.product-purchase-features-list__item .purchase-detail {
+.product-purchase-features-list__item .purchase-detail, .product-purchase-features-list__item-hc {
 	background-color: $white;
 	border-radius: 0;
 	margin: 0;
 }
+
+.product-purchase-features-list__item-hc {
+	padding: 32px;
```

Comment:
This could easily be applied to `.product-purchase-features-list__item .happiness-support`, right?

Reply:
Changed as per comment above. Thanks!